modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -60,6 +65,13 @@ public void testExportImportObjectDefinition() throws Exception {
 		testExportImport(
 			"test-object-definition.json", "test-object-definition.json", null,
 			"TestObjectDefinition");
+
+		JSONObject actualJSONObject = _baseObjectDefinitionJSONObject;
```

Comment:
In this case, i think we can skip the assignments to new variables and just do an import test using the `_baseObjectDefinitionJSONObject` directly